docker-compose.yml

```
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  nginx:
+    image: nginx:latest
+    container_name: production_nginx
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    ports:
+      - 8080:80
+    depends_on:
+      - meilisearch
+
+  meilisearch:
+    image: getmeili/meilisearch:latest
```

We could use this https://docs.docker.com/compose/compose-file/compose-file-v3/#variable-substitution

```suggestion
    image: "getmeili/meilisearch:${MEILISEARCH_VERSION}"
```

And create a file `.env` with `MEILISEARCH_VERSION=latest`.

According to the compose docs, the `.env` will be used when they cannot find the MEILISEARCH_VERSION env var defined, so this handles the local environment usage and for the tests.yml script.

And in the pre-release script I think this works: `run: MEILISEARCH_VERSION=${{ env.MEILISEARCH_VERSION }} docker-compose up -d` or just `run: docker-compose up -d`

---

It's fixed